### DISCUSSION TITLE
How to run Minio as a background process?

### DISCUSSION THREAD

**Discussion by stevekm** [minio/minio#11694] [Q&A] [answered]
How do you run Minio server as a background process?

The Issue [here](https://github.com/minio/minio/issues/4155) has descriptions on how to run Minio server as a systemd service, but that is not helpful because I do not have access to systemd on the server. For example;

```
$ cd /etc/systemd/system/

$ touch foo
touch: cannot touch ‘foo’: Permission denied
```

I can run plenty of other services as a normal, unpriveledged user, such as PostgreSQL, RabbitMQ, nginx, Celery, etc.. How do I do this with Minio server as well?

**Comment by harshavardhana** [comment, marked as answer]
If you don't have access to systemd just run it like 

```
minio server args &
```
